clients/privacy-center/packages/fides-consent/README.md

```diff
@@ -85,24 +111,56 @@ if (Fides.consent.data_sales) {
 }
 ```
 
+### Meta Pixel
+
+
+To integrate with [Meta's tracking pixel](https://developers.facebook.com/docs/meta-pixel/get-started),
+call `Fides.meta(options)`, where options is an with the following properties:
```

Comment:
@ssangervasi - this says "options is an with the following properties:"  Is it an object?